### PR TITLE
MMT-891

### PR DESCRIPTION
**Pull request by macrouch**

### CODE REVIEW

File: app/controllers/drafts_controller.rb [hunk 1]
```diff
@@ -21,6 +21,7 @@ def show
     set_country_codes
     set_language_codes
     @errors = validate_metadata
+    logger.info @errors
```
Comment (macrouch) [inline app/controllers/drafts_controller.rb]: remove this

**Comment by macrouch** [comment]
check the required urls required fields progress circle

File: app/controllers/drafts_controller.rb [hunk 2]
```diff
@@ -298,18 +299,49 @@ def validate_picklists(errors, metadata)
         end
       end
 
+      # TODO this needs to happen every time RelatedUrls shows up in the metadata
```
Comment (macrouch) [inline app/controllers/drafts_controller.rb]: todo

**Comment by abbottry** [comment]
How important is it that the labels match the title of the help modal? This is for `Content Type` but the modal says `Content`. 

Related-ish, I remember discussing the `U R L` and WHY it exists, but I thought you said you had a work around for it to prevent it from displaying like that?

screen shot 2017-05-12 at 10 06 32 am

**Comment by abbottry** [comment]
Using the help modals, I'm getting the incorrect (previous) modal a fair amount of times.

**Comment by macrouch** [comment]
@abbottry I don't have anything for the U R L part, I just know it has been like that the entire time we have had help modals

I'll look into getting Type to show up

Which incorrect modals are you seeing? Any javascript errors when you do?

**Comment by abbottry** [comment]
Ignore all of these 😞 , somehow.. i didn't have the latest because I had un-staged changes. Things are working fine for me now, will continue my review with.. your legit, newest code. :man_facepalming:

**Comment by macrouch** [comment]
haha, I also see U R L Content though, so I still want to find out why Type isn't there

**Pull request merged**